### ISSUE TITLE
Review accessibility of audio player component

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#604]
Review HTML structure of audio player component for accessibility.

Siteimprove report for label issue in audio player component:  https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=64779238728&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone#/sia-r8/failed/ai9lMGJNakw=;M2ZmYjExM2M=.
